### PR TITLE
(node/antu[01-04].ls.lsst.org) cluster configuration

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

**Review by jhoblitt — CHANGES_REQUESTED** [review]
The chassis model needs to be added to https://github.com/lsst-it/lsst-control/blob/production/site/profile/manifests/core/hardware.pp#L18 to configure the bmc.

File: spec/hosts/nodes/antu01.ls.lsst.org_spec.rb [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      context 'with enp65s0f1' do
+        let(:interface) { 'enp65s0f1' }
+
+        it_behaves_like 'nm enabled interface'
+        it_behaves_like 'nm dhcp interface'
+        it_behaves_like 'nm ethernet interface'
+      end
+
+      context 'with enp65s0f0' do
+        let(:interface) { 'enp65s0f0' }
```
Comment (jhoblitt) [inline spec/hosts/nodes/antu01.ls.lsst.org_spec.rb]: I'm guessing this is the trunk interface?  If so, it should be in the list of disabled interfaces (we do it that way as we don't want either a static or dhcp address assigned to the interface).

File: site/profile/manifests/core/hardware.pp [hunk 2]
```diff
@@ -42,8 +42,10 @@
     /SSG-640SP-E1CR90/: {
       include ipmi
     }
+    /1115HS-TNR/: {
+      include ipmi
+    }
   }
-
```
Comment (jhoblitt) [inline site/profile/manifests/core/hardware.pp:46]: Why is this whitespace being removed?

Comment (dtapiacl) [reply site/profile/manifests/core/hardware.pp:46]: Typo.

File: spec/hosts/nodes/antu01.ls.lsst.org_spec.rb (hunk 1 above)
Comment (dtapiacl) [reply spec/hosts/nodes/antu01.ls.lsst.org_spec.rb]: Added the interface, to the disabled list.

**Review by jhoblitt — COMMENTED** [review]
This code looks good but I don't see IP assignments for these hosts in ipam.  Could you add those?

**Review by jhoblitt — APPROVED** [review]
Thank you for updating ipam!

**Pull request merged**